sok/management/commands/snowball.py

```diff
@@ -155,6 +164,75 @@ def handle_objs(
 						self.echo(abstract)
 				elif choice in {'', 'n', 'no'}:
 					# TODO Import?
+					data = semanticscholar(paper_id)
+					# Add authors to database
+					authors: List[Author] = []
+					first = True
+					cite_key = ''
+					for author in data.get('authors', []):
+						name = author.get('name', '')
```

Comment:
It makes no sense adding authors with empty names. A better approach might be:

```suggestion
						if name := author.get('name', None):
						        …
```